reference/configuration/tidb-server/configuration-file.md

```diff
@@ -378,7 +378,7 @@ prepare 语句的 Plan cache 设置。
 
 ### `enable-chunk-rpc`
```

Comment:
Is this option worth being public? It breaks `Don't Make Me Think` law. 
Users would touch this option only when they met the occurrence of bug. IMO, this option could stay in the environment variable and be retired sometime.

Reply:
@zz-jason PTAL, too.

Reply:
agree